evm/src/witness/memory.rs

```diff
@@ -86,6 +86,18 @@ pub struct MemoryOp {
     pub value: U256,
 }
 
+pub static DUMMY_MEMOP: MemoryOp = MemoryOp {
+    filter: false,
+    timestamp: 0,
+    address: MemoryAddress {
+        context: 0,
+        segment: 0,
+        virt: 0,
+    },
+    kind: MemoryOpKind::Read,
+    value: U256::zero(),
+};
```

Comment:
Yeah, I agree, but I didn't want to change the type of all returned pairs in a call like `[(in0, _), (in1, log_1), (in2, log2), (in3, log3)] = stack_pop_with_log_and_fill::<4, _>` just because of the first one.
I got rid of it anyway in a following PR by returning the "correct" log if the top of the stack was stored in the Stack segment (which is useful in one particular case relevant to the PR). I can cherry-pick this change here if you want?